pkg/network/ebpf/c/tracer/tracer.h

```diff
@@ -54,6 +60,7 @@ typedef struct {
     protocol_stack_t protocol_stack;
     __u8 flags;
     __u8 direction;
+    tls_info_t tls_tags;
```

Comment:
what's the cost for this addition?

Reply:
there shouldn't be any cost as this takes up the space previously used by cgo padding. If it increased the size of the struct we would go over the eBPF stack size limit (due to the known issues in closed connection batching)

Reply:
the cost is going to be reflected via the ebpf maps rss